packages/commercetools/api-client/src/index.ts

```diff
@@ -53,17 +53,26 @@ const setup = <TCacheShape>(setupConfig: SetupConfig<TCacheShape>): ApolloClient
   return apolloClient;
 };
 
+const tokenChangeListeners = [];
+
+const onTokenChange = (fn: (token) => void): void => {
+  tokenChangeListeners.push(fn);
+};
+
+const tokenChanged = (newToken) => tokenChangeListeners.forEach(fn => fn(newToken));
```

Comment:
IMO these functions are named actually the same and it's not readable because of that.
What do you think to rename the second one into: `changeToken(newToken)` ?

Also could you add missing types? (newToken I guess is string)